concourse/steps/update_component_deps.py

```diff
@@ -562,10 +551,22 @@ def create_upgrade_pr(
             split_release_notes = [release_notes]
 
         if len(split_release_notes) > 1:
-            pr_body += max_body_length_exceeded_remark
-            additional_notes = split_release_notes
+            pr_body += split_release_notes[0] + max_body_length_exceeded_remark
+            additional_notes = split_release_notes[1:]
```

Comment:
what if the additional_notes block also exceeds the size limit?

Reply:
does `step_size` not already ensure that the chunks in `split_release_notes` are wihtin the size limit? since `step_size` is calculated based on `max_pr_body_length`, each chunk in `additional_notes` should already respect the max allowed size. am i wrong?